src/pages/member/images/house.svg

```diff
@@ -0,0 +1,22 @@
+<svg width="256" height="208" viewBox="0 0 256 208" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g opacity="0.75">
+<rect x="68.5415" y="53.27" width="27.4412" height="94.8973" rx="1.87915" fill="#00C614"/>
+<path d="M125.503 44.8137L184.543 115.752H68.5415L125.503 44.8137Z" fill="#96FF50"/>
```

Comment:
Wanna try using an svg minifier to reduce the file size a bit?

Reply:
didnt know that was a thing, that's pretty cool - will look into it

Reply:
![image](https://user-images.githubusercontent.com/45786074/85798403-438ee300-b6f2-11ea-8321-acc6c359195c.png)
 
big W

Reply:
.4 kB we out here (make sure to do it for the other svgs)